setup.py

```diff
@@ -59,7 +58,7 @@
     # your project is installed. For an analysis of "install_requires" vs pip's
     # requirements files see:
     # https://packaging.python.org/en/latest/requirements.html
-    install_requires=["web3", "psycopg2", "click", "attrs"],
+    install_requires=["web3>=4.4.1,<4.5", "psycopg2>=2.7,<2.8", "click", "attrs"],
```

Comment:
whats the problem with psycopg2?

Reply:
they will release a dedicated psycopg2 binary and non binary release when 2.8 comes. Though I don't expect any of our code to break.